server/Dockerfile

```diff
@@ -0,0 +1,70 @@
+#* ###########################
+#* ##### Stage 0: Base #######
+#* ###########################
+FROM node:12.18.1-buster-slim AS base
+
+# Ensure container is up to date
+RUN apt-get update && apt-get install --no-install-recommends --yes openssl
+
+# Establish working directory to contain application
+WORKDIR /app
+
+#* ###########################
+#* #### Stage 1: Builder #####
+#* ###########################
+FROM base AS builder
+COPY package.json yarn.lock ./
+
+# Install Production Dependencies
+RUN yarn install --production --pure-lockfile
+RUN mkdir tmp && cp -a ./node_modules/. ./tmp/node_modules/
+
+# Install Dev Dependencies
+RUN yarn install --pure-lockfile
+
+# Run Migrations
+# Note: Uncomment this step if you want to run migrations
+#       every time you build a new production version
+# COPY prisma/migrations/ ./prisma/migrations/
+# RUN npx prisma migrate up --experimental
+
+# Generate Prisma Client
+COPY prisma/schema.prisma ./prisma/schema.prisma
+RUN npx prisma generate
+
+# Copy source code
+COPY /src/ ./src/
+
+# Note: You don't seem to have a build script at the moment
+#       If/ when you add one uncomment this step
+# RUN yarn build
+
+#* ###########################
+#* ##### Stage 2: Runner #####
+#* ###########################
+FROM base AS runner
```

Comment:
dont we want to have base and rest of the app separated? Or have it as one.
We do not need to rebuild all if we do not update the base on every build?